x-pack/plugins/observability_solution/observability_ai_assistant_app/public/components/rca/rca_timeline/index.tsx

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { AreaSeries, Axis, Chart, Position, ScaleType, Settings } from '@elastic/charts';
+import { EuiPanel, EuiSkeletonText } from '@elastic/eui';
+import type { SignificantEventsTimeline } from '@kbn/observability-ai-server/root_cause_analysis';
+import moment from 'moment';
+import React, { useMemo, useRef } from 'react';
+import { useChartTheme } from '../../../hooks/use_chart_theme';
+import { SignificantEventAnnotation } from './significant_event_annotation';
+import { TIME_LINE_THEME } from './timeline_theme';
+
+interface RootCauseAnalysisTimelineProps {
+  start: number;
+  end: number;
+  timeline: SignificantEventsTimeline;
+  loading: boolean;
+}
+
+const InnerRootCauseAnalysisTimeline = ({
+  start,
+  end,
+  loading,
+  timeline,
+}: RootCauseAnalysisTimelineProps) => {
+  const baseTheme = useChartTheme();
+
+  const chartRef = useRef(null);
+
+  const data = useMemo(() => {
+    const points = [
+      { x: moment(start).valueOf(), y: 0 },
+      { x: moment(end).valueOf(), y: 0 },
+    ];
+
+    // adding 100 fake points to the chart so the chart shows cursor on hover
+    for (let i = 0; i < 100; i++) {
+      const diff = moment(end).valueOf() - moment(start).valueOf();
+      points.push({ x: moment(start).valueOf() + (diff / 100) * i, y: 0 });
+    }
+    return points;
+  }, [start, end]);
```

Comment:
Is this a workaround for a known issue with elastic charts?

Reply:
I copied this over from the Investigate app - it's not being used, will remove.